benchmarks/buiter_2016_wedge/buiter_2016_wedge.prm

```diff
@@ -0,0 +1,172 @@
+# This is the input file for the setup for the benchmark described in Buiter et al, 2016
+# Experiment 1 which simulates a stable wedge undergoing translation.
+
+
+#####  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 14400
+set Use years in output instead of seconds = false
+set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Nonlinear solver tolerance             = 1e-9
+set Max nonlinear iterations               = 25
+set CFL number                             = 0.5
+set Maximum time step                      = 1e6
+set Output directory                       = output
+set Timing output frequency                = 1
+set Pressure normalization                 = no
+
+#### Parameters describing the model
+
+# Model geometry
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X repetitions = 500
+    set Y repetitions = 100
+    set X extent      = 0.15
+    set Y extent      = 0.035
```

Comment:
The model geometry and number of repetitions will give slightly different spacing in the vertical direction. Can you change them to get the same vertical and horizontal grid spacing? Also, somewhere in the documentation (parameter file, README file) you will want to describe how varying the resolution affects the results.